doc/arkode/guide/source/Usage/MRIStep/Custom_Inner_Stepper/Description.rst

```diff
@@ -239,6 +228,61 @@ Setting Member Functions
    **Example codes:**
       * ``examples/arkode/CXX_parallel/ark_diffusion_reaction_p.cpp``
 
+.. c:function:: int MRIStepInnerStepper_SetAccumulatedErrorGetFn(MRIStepInnerStepper stepper, MRIStepInnerGetAccumulatedError fn)
+
+   This function attaches an :c:type:`MRIStepInnerGetAccumulatedError` function to an
+   :c:type:`MRIStepInnerStepper` object.
+
+   :param stepper: an inner stepper object.
+   :param fn: the :c:type:`MRIStepInnerGetAccumulatedError` function to attach.
+
+   :retval ARK_SUCCESS: if successful
+   :retval ARK_ILL_INPUT: if the stepper is ``NULL``
+
+   **Example usage:**
```

Comment:
I am not sure that these code blocks (the one I am commenting on and subsequent) add much. I think they would need to show more context to be worth having.